.vscode/tasks.json

```diff
@@ -16,7 +16,7 @@
     {
       "label": "launch-dev",
       "type": "shell",
-      "command": "eval $(minikube docker-env --shell bash); yes | pachctl undeploy; LAUNCH_DEV_ARGS='--cluster-deployment-id=dev' make launch-dev",
+      "command": "eval $(minikube docker-env --shell bash); make clean-launch; LAUNCH_DEV_ARGS='pachd.clusterDeploymentID=dev' make launch-dev",
```

Comment:
Moved the windows launch fixes from #6556 because that won't be merged for a while and this is impacting my development flow.  cc @chainlink in case you have opinions on these changes because it's related to the helm change.